pkg/trace/interpreter/interpreters/default_span_interpreter_test.go

```diff
@@ -0,0 +1,35 @@
+package interpreters
+
+import (
+	"github.com/StackVista/stackstate-agent/pkg/trace/interpreter/config"
+	"github.com/StackVista/stackstate-agent/pkg/trace/pb"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDefaultSpanInterpreter(t *testing.T) {
+	for _, tc := range []struct {
+		testCase    string
+		interpreter *DefaultSpanInterpreter
+		span        pb.Span
+		expected    pb.Span
+	}{
+		{
+			testCase:    "Should not change the service name if there are no matching identifiers",
+			interpreter: MakeDefaultSpanInterpreter(&config.Config{}),
+			span:        pb.Span{Service: "SpanServiceName", Meta: map[string]string{"some.meta": "MetaValue"}},
+			expected:    pb.Span{Service: "SpanServiceName", Meta: map[string]string{"span.serviceName": "SpanServiceName", "some.meta": "MetaValue", "span.serviceURN": "urn:service:/SpanServiceName"}},
+		},
+		{
+			testCase:    "",
```

Comment:
can you fill the name of this test case?

Reply:
Done